locale/core.yml

```diff
@@ -500,6 +500,7 @@ core:
   # Translations in this namespace are used by the forum and admin interfaces.
   lib:
     debug_button: Debug
+    close_button: => core.ref.close
```

Comment:
I'd prefer this to be listed under a `modal` key to ensure we don't use this in other components too. We try to keep to separate translation keys for each instance of text.